src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -5,15 +5,16 @@
       "longName": "no-restore",
       "shortName": ""
     },
-    "PWA": {
-      "longName": "pwa",
-      "isHidden": true
-    },
     "UseServer": {
       "longName": "use-server"
     },
     "UseWebAssembly": {
-      "longName": "use-wasm",
+      "longName": "use-wasm"
+    },
+    "Empty": {
+      "longName": "empty"
+    },
+    "IncludeSampleContent": {
```

Comment:
Do not block on this, but we were talking in the past of making `blazor web` bring in server and webassembly by default. 

The reason we did opt-in as opposed to opt-out was that we didn't have the underlying features in place. Do we want to keep that model (opt-in) or do we want to switch to opt-out.

/cc: @danroth27

Reply:
Dan and I did discuss this and came to the conclusion that server-on-by-default, wasm-off-by-default was the preferred balance for .NET 8.

Reply:
@javiercn I want interactivity to work by default, but I'm concerned about the complexity that WebAssembly currently adds. I'm thinking about the getting started experience here for new users. Granted, the downside of having WebAssembly off by default is that if you later want to add it, it's not easy to do.